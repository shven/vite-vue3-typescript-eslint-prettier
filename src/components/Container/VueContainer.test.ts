import { mount } from '@vue/test-utils';
import VueContainer from './VueContainer.vue';

describe('VueContainer.vue', () => {
    const wrapper = mount(VueContainer, {
        slots: {
            default: 'Slot content goes here'
        }
    });

    it('renders slot content', () => {
        expect(wrapper.text()).toContain('Slot content goes here');
    });

    it('Match snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
