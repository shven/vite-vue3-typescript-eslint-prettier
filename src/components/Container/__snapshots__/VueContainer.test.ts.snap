// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VueContainer.vue > Match snapshot 1`] = `
<div
  class="container"
  data-v-cbb0d1af=""
>
  
  Slot content goes here
  
</div>
`;
